pkgs/development/compilers/openjdk/openjfx/11.nix

```diff
@@ -87,24 +88,20 @@ in makePackage {
   postFixup = ''
     # Remove references to bootstrap.
     find "$out" -name \*.so | while read lib; do
-      new_refs="$(patchelf --print-rpath "$lib" | sed -E 's,:?${bootjdk}[^:]*,,')"
+      new_refs="$(patchelf --print-rpath "$lib" | sed -E 's,:?${openjdk11-bootstrap}[^:]*,,')"
       patchelf --set-rpath "$new_refs" "$lib"
     done
-
-    # Test to make sure that we don't depend on the bootstrap
```

Comment:
I remember faintly that for some reason `disallowedReferences` didn't work for me when I wrote this. Then again, I don't see _why_ should they fail and I failed to properly document the reason. If it builds it should work so let's keep it that way.